pkgs/data/themes/catppuccin-gtk/default.nix

```diff
@@ -52,33 +52,26 @@ stdenvNoCC.mkDerivation rec {
 
   propagatedUserEnvPkgs = [ gtk-engine-murrine ];
 
-  patches = [
-    # Allows installing with `-t all`. Works around missing grey assets.
-    # https://github.com/catppuccin/gtk/issues/17
-    ./grey-fix.patch
-  ];
-
-  postPatch = ''
-    patchShebangs --build scripts/*
-    substituteInPlace Makefile \
-      --replace '$(shell git rev-parse --show-toplevel)' "$PWD"
-    substituteInPlace 'scripts/install.sh' \
-      --replace '$(git rev-parse --show-toplevel)' "$PWD"
+  prePatch = ''
```

Comment:
This should be `postPatch`. Was there a reason you moved it to `prePatch`, did something break?

Reply:
You're right, nothing breaks. I don't remember why I had it in `prePatch`.